state-chain/chains/src/lib.rs

```diff
@@ -634,9 +718,18 @@ pub enum TransferFallbackError {
 	Unsupported,
 	/// Failed to lookup the given token address, so the asset is invalid.
 	CannotLookupTokenAddress,
+	/// Some other DispatchError occurred.
+	DispatchError(DispatchError),
 }
+
 pub trait TransferFallback<C: Chain>: ApiCall<C::ChainCrypto> {
-	fn new_unsigned(transfer_param: TransferAssetParams<C>) -> Result<Self, TransferFallbackError>;
+	fn new_unsigned(transfer_param: TransferAssetParams<C>) -> Result<Self, TransferFallbackError> {
```

Comment:
It would be good to have doc comments on these default methods mentioning that they should not be overridden, and the `_impl` method should be implemented instead.